src/Command/PluginLoadCommand.php

```diff
@@ -109,6 +109,14 @@ protected function modifyApplication(string $app, string $plugin): void
             $this->abort();
         }
 
+        // Check if plugin is already loaded
+        $regex = '/ \$this->addPlugin\(\'' . $plugin . '\'\);$/mu';
```

Comment:
Would we be able to use the PluginRegistry to see if the plugin is already loaded? This looks like it could fail if the plugin was loaded with config options.

Reply:
At least the closing bracket should not be part of the regex

Reply:
yup, definitely didn't think about plugins with config - just what we generate by default.